### PR TITLE
[Selenium] Set application.confirmExit property to 'never' by default

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
Set application.confirmExit property to never as default.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/16100

### CODE REVIEW

**Comment by SkorikSergey** [comment]
[ci-test]

**Comment by SkorikSergey** [comment]
[ci-build]

**Comment by SkorikSergey** [comment]
[ci-test]

**Comment by SkorikSergey** [comment]
[ci-test]

**Comment by SkorikSergey** [comment]
[ci-test]

**Comment by SkorikSergey** [comment]
crw-ci-test

File: tests/legacy-e2e/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/client/TestUserPreferencesServiceClient.java [hunk 1]
```diff
@@ -36,9 +37,13 @@
 
   @Inject
   public TestUserPreferencesServiceClient(
-      TestApiEndpointUrlProvider apiEndpointProvider, HttpJsonRequestFactory httpRequestFactory) {
+      TestApiEndpointUrlProvider apiEndpointProvider, HttpJsonRequestFactory httpRequestFactory)
+      throws Exception {
     this.apiEndpoint = apiEndpointProvider.get().toString();
     this.httpRequestFactory = httpRequestFactory;
+
+    //
```
Comment (dmytro-ndp) [inline tests/legacy-e2e/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/client/TestUserPreferencesServiceClient.java]: please, add comment about why do we need to set properties here, and what that property is about

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**